cypress/integration/logout.spec.js

```diff
@@ -0,0 +1,16 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('User can logout', () => {
+  autoRecord();
+
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('when click in logout redirects user to login', function () {
```

Comment:
@rafayet-monon  How about using `describe` to make it easier to read 🙌🏼 

```suggestion
  describe('when click on logout link', () => {
    it('redirects user to login page', () => {
```